### DISCUSSION TITLE
Github releases

### DISCUSSION THREAD

**Discussion by maratori** [lona-web-org/lona#99] [Feature Request]
Let's create github releases as well as pypi releases.

I found them very useful in several aspects.
The most interesting is that users can use github notifications to get email on each new release.
It isn't possible if you create git tag only (without explicit github release).

**Comment by fscherf** [comment]
OK. Could you provide some documentation on how to do this properly (a comment here is enough)? At this point we should add an article to the documentation on how to make an release and how to update the documentation.

**Comment by maratori** [comment]
I think github [documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) is clear enough. But if you have any question I'd be happy to answer.

Here is a great [example](https://github.com/plotly/dash/releases) on how releases should look like.

**Comment by fscherf** [comment]
Oh! I should have googled that myself :D thanks for the links!
Do you want me to upload all previous releases, or should we do this up this point?

**Comment by maratori** [comment]
It's up to you.

**Comment by fscherf** [comment]
@maratori I created a github release for 1.6.1 and i will release them here from now on. I am not sure if it was clever to start with the latest release, because i think i will mess up the order of release tags on github if start to upload earlier releases

**Reply by maratori** [reply]
I hope github is smart enough to keep correct order. Each release has tag, so it is possible.
Also it should be noted that even without explicit releases, github shows tags here https://github.com/lona-web-org/lona/releases

**Reply by fscherf** [reply]
Ok. Then i will try to upload all releases back to 1.0

**Reply by fscherf** [reply]
@maratori Done!
